c/bobyqa_c.f90

```diff
@@ -47,12 +48,16 @@ subroutine bobyqa_c(cobj_ptr, data_ptr, n, x, f, xl, xu, nf, rhobeg, rhoend, &
 integer(IK) :: npt_loc
 integer(IK) :: nf_loc
 real(RP) :: f_loc
-real(RP) :: rhobeg_loc
-real(RP) :: rhoend_loc
+real(C_DOUBLE), pointer :: rhobeg_loc_intermediate
```

Comment:
Is there a need to shorten the variable name? It's only used in 2 lines which do not overflow. If there is a need for shorter variable names, i.e. because they are used often and cause many lines to overflow, then I would prefer using synonyms that are shorter as opposed to artificially shortening the word. As I mentioned before, I did not know for a long time after starting to work with this codebase that `_loc` was short for `_local`. I thought `_loc` was just an arbitrary suffix, and so I missed out on the semantic meaning. That is why I prefer using full words as opposed to artificially shorted ones (exceptions can be made of course, and ptr is a good example, but they should be rare).

Reply:
It is mainly because `_loc` and `_intermediate` are inconsistent in terms of style.  

If `_loc` was sprlled as `_local`, then I would not have raised this point.

Reply:
I still think `interm` takes away too many letters, how about `intrmdiate`?